inc/ThirdParty/Plugins/Ecommerce/WooCommerceSubscriber.php

```diff
@@ -513,6 +513,9 @@ private function get_nonce_actions() {
 	 */
 	private function product_has_gallery_images() {
 		$product = wc_get_product( get_the_ID() );
+		if ( ! $product || empty( $product ) ) {
```

Comment:
empty will check for both null and false, so we don't need the first part of the condition